src/pubg-dotnet/Infrastructure/Region.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Pubg.Net.Infrastructure
```

Comment:
Since this will be used by the end user, have this under the Pubg.Net namespace. I'd like people to just have to use one namespace and have access to everything they need to make a request. Maybe also move this from the infrastructure folder to Models/Common

Reply:
Yeah I wasn't sure where to put it. I'll fix it right now.